File: WebTotalCommander/WebTotalCommander.Service/ViewModels/Common/FolderGetAllViewModel.cs
using WebTotalCommander.FileAccess.Utils;

namespace WebTotalCommander.Service.ViewModels.Common;

public class FolderGetAllViewModel
{
    private List<FolderFileViewModel> _fileFolders;

    public List<FolderFileViewModel> FolderFile
    {
        get => _fileFolders ??= new List<FolderFileViewModel>();
        set => _fileFolders = value;
    }
    public PaginationMetaData PaginationMetaData { get; set; } = new PaginationMetaData();
}

File: WebTotalCommander/webtotalcommander.client/src/app/services/models/common/folder_file_getall/folder.getall.view-model.ts
import { FolderFileViewModel } from "./folder.file.view-model";
import { PaginationMetaDataView } from "./pagination.data";

export class FolderGetAllViewModel {
    public folderFile: Array<FolderFileViewModel> = [];
    public paginationMetaData: PaginationMetaDataView = new PaginationMetaDataView();
}